vuepress/docs/next/tutorials/create/mfe/angular.md

```diff
@@ -207,43 +207,38 @@ You should see a reponse after a few moments:
 You now have an Angular micro frontend running.
 :::
 
-## Build the Bundle and Install to Entando
+## Build, Publish and Install the Bundle
 
 ### Build and Publish the Bundle 
 
-1. Open `entando.json` in the bundle root folder and add the following to the `angular-widget` parameters. 
+1. Open `entando.json` in the bundle root folder and add the following under the `angular-widget` parameters. 
 ```json
 "buildFolder": "dist/angular-widget"
-
 ```
-2. Add a thumbnail JPEG or PNG file to the root bundle folder. It must be named thumbnail and not exceed 100kB, e.g. thumbnail.png
+2. (Optional) Add a thumbnail JPEG or PNG file to the root bundle folder. It must be named `thumbnail` and not exceed 100kB, e.g. thumbnail.png.
 
-3. Build the components
-```sh
-ent bundle build --all
-```
-4. Generate the Docker image for the bundle 
+3. Pack the bundle which includes the build and generate Docker images steps:
```

Comment:
maybe add a comma before "which" or put "which..." in parentheses; this is useful info but a side note and independent of the instruction

Reply:
that looks odd when I tried it.